arrow-site/docs/_data/doc-versions.yml

```diff
@@ -1,34 +1,25 @@
 docVersions:
 
-  - title: "v0.10.5"
-    url: https://arrow-kt.io/docs/0.10/
-    previous: https://arrow-kt.io/docs/0.9/
-
-  - title: "v0.11.0"
-    url: https://arrow-kt.io/docs/0.11/
-    previous: https://arrow-kt.io/docs/0.10/
-
-  - title: "v0.12.1"
+  - title: "Version 0.12.x"
     url: https://arrow-kt.io/docs/0.12/
     previous: https://arrow-kt.io/docs/0.11/
```

Comment:
```suggestion

```

It feels a bit strange to be able to see the docs of a version which is not in the list.

Reply:
Yes. you're right. And probably, we don't need that information anymore, as we have removed the `Previous` link`